### PR TITLE
Add return URL back to Canvas on reset email

### PR DESCRIPTION
**Pull request by tpaulshippy**
I am unable to test this in my current environment, so this may be horribly wrong. But I think this will help with some of the scenarios around users getting lost after they are created. This does depend on a bug fix being deployed in Identity (https://dev.azure.com/strongmind/Strongmind/_git/Identity/pullrequest/6942)

### CODE REVIEW

File: app/decorators/models/user_decorator.rb [hunk 1]
```diff
@@ -226,6 +226,7 @@ def validate_identity_creation
         "FirstName" => first_name,
         "LastName" => last_name,
         "Email" => identity_email,
+        "PasswordResetReturnUrl" => "http://#{ENV['CANVAS_DOMAIN']}",
```
Comment (YupItsZac) [inline app/decorators/models/user_decorator.rb:229]: This should almost certainly be HTTPS

**Comment by YupItsZac** [comment]
Aside from a that single (not critical) comment above, this looks solid. I’d like @AndyRosenbergStrongMind specifically  to take a look though since he’s out Rubyist

File: app/decorators/models/user_decorator.rb (hunk 1 above)
Comment (tpaulshippy) [reply app/decorators/models/user_decorator.rb]: Done

**Review by AndyRosenbergStrongMind — COMMENTED** [review]
I'll test this on NewIdSandBox

**Review by AndyRosenbergStrongMind — APPROVED** [review]
Works

**Pull request merged**